src/main/java/com/puppycrawl/tools/checkstyle/checks/metrics/AbstractClassCouplingCheck.java

```diff
@@ -72,6 +72,8 @@
             "List", "ArrayList", "Deque", "Queue", "LinkedList",
             "Set", "HashSet", "SortedSet", "TreeSet",
             "Map", "HashMap", "SortedMap", "TreeMap",
+            // Annotations
+            "Override", "Deprecated", "SafeVarargs", "SuppressWarnings",
```

Comment:
Good question raised: https://github.com/xwiki/xwiki-commons/pull/20#discussion_r140438164
I think we should always exclude these annotations like we do for imports with `java.lang`

Reply:
@Vladlis, Why we exclude only annotations, but not primitive type, exception and etc. ?
@romani , What do you think?

Reply:
Actually I find this strange - if user needs to add one more class to excludes he also has to duplicate all our defaults which I guess are usually excluded by everyone, so this leads to a huge list of classes in config.